index.html

```diff
@@ -13,90 +18,66 @@
     <title>판다마켓</title>
   </head>
   <body>
-    <!-- 헤더 -->
-    <header class="header">
+    <header class="header fixed">
       <div class="header-container">
         <a href="/" class="logo">
-          <img src="public/logos/Property 1=lg.png" alt="판다로고" />
+          <img src="public/logos/panda-market-logo.png" alt="판다로고" />
         </a>
 
         <a href="login.html" class="login-btn">로그인</a>
       </div>
     </header>
-    <!-- 메인 -->
     <main>
-      <!-- 섹션1 -->
-      <section class="main-banner">
-        <div class="main-banner-container">
-          <div class="main-banner-text">
-            <h1>일상의 모든 물건을<br />거래해 보세요</h1>
-            <a href="items.html" class="action-btn">구경하러 가기</a>
-          </div>
-          <div class="main-banner-image">
-            <img src="public/images/Img_home_top.png" alt="메인배너" />
-          </div>
+      <section class="main-banner banner" id="hero">
+        <div class="wrapper">
+          <h1>일상의 모든 물건을<br />거래해 보세요</h1>
+          <a href="items.html" class="action-btn button pill-button"
+            >구경하러 가기</a
+          >
```

Comment:
💊 제안
현재 코드에서는 하나의 태그가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~


```suggestion
          <a href="items.html" class="action-btn button pill-button">
            구경하러 가기
          </a>
```